lvm-cache-1.sh

```diff
@@ -17,7 +17,8 @@
 #
 # Red Hat Author(s): Chris Lumens <clumens@redhat.com>
 
-TESTTYPE="lvm storage"
+# FIXME: Anaconda crashes with BlockDev.LVMError
```

Comment:
Could you please add a link to https://bugzilla.redhat.com/show_bug.cgi?id=1886767?

Reply:
Nice that it's already supported! I adjusted the comment and commit message accordingly.